config/rootcompute/kube-1.24/apiexport-kubernetes.k8s.io.yaml

```diff
@@ -11,6 +11,7 @@ spec:
     - v124.services.core
     - v124.deployments.apps
     - v124.pods.core
+    - v124.endpoints.core
```

Comment:
what about EndpointSlices? Does one need them? Does one need both or do endpoints always exist?

Reply:
For now we didn't have any use-case requiring upsyncing the `EndpointSlice`s.
This PR tackles minimal requirements spotted by the KNative support scenario.
Additional resources will be added in the future as required.

It would still be possible to import `EndpointSlice`s from the physical cluster if needed, and sync them as any other resource if necessary.